packages/prerender-proxy/README.md

```diff
@@ -7,7 +8,13 @@ The `prerender` will function also make a HEAD request to a nominated backend to
 
 These functions are intended to be added to an existing Cloudfront  
 
+## Cache Control
+
+The intention of the `cache-control` function is to avoid/control CloudFront Caches for `prenrender` bot.
+This function to be associated with CloudFront's`origin response`
+
 ## Props
+
 `redirectBackendOrigin`: The backend origin to make the HEAD request to.
 `redirectFrontendHost`: This hostname is used to replace the backend host for any redirects that contain the backend host.
 `prerenderToken`: Your prerender.io authentication token 
```

Comment:
Can you add the new env variables to the README? `PRERENDER_CACHE_KEY`, `PRERENDER_CACHE_MAX_AGE` and add a note that they are optional.